docs/reference/esql/functions/log.asciidoc

```diff
@@ -0,0 +1,36 @@
+[discrete]
+[[esql-log]]
+=== `LOG`
+
+*Syntax*
+
+[.text-center]
+image::esql/functions/signature/log.svg[Embedded,opts=inline]
+
+`base`::
+Numeric expression. If `null`, the function returns `null`.
+
+`exponent`::
```

Comment:
I think `exponent` is a bit confusing in the context of logarithm (the exponent is supposed to be the result...). Maybe `value` is good enough (and I see it's like that also in SHOW FUNCTIONS)

Reply:
Thank you for catching the typo!